vaadin-grid.html

```diff
@@ -457,7 +459,8 @@
   .vaadin-grid-header th,
   .vaadin-grid-body td,
   .vaadin-grid-footer td {
-       padding: 0 24px 0 24px;
+    @apply(--vaadin-grid-row-cell);
```

Comment:
You should move this below the padding style, that way padding can be overridden in the mixin without the use of !important.